composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "yotpo/module-yotpo-core",
     "description": "Yotpo Reviews core extension for Magento2",
-    "version": "4.0.18",
+    "version": "4.1.0.6",
```

Comment:
Please drop this commit - 841e84afd704439cf6f00a8af4e8c6282a7f8822.
In case you need a version bump, just open a new branch just to bump the version.

Reply:
Will be done after sanity testing.